cli/default-reporter/src/reporterForClient/reportSummary.ts

```diff
@@ -119,6 +121,9 @@ function printDiffs (
     if (pkg.from) {
       result += ` ${chalk.grey(`<- ${pkg.from && path.relative(opts.prefix, pkg.from) || '???'}`)}`
     }
+    if (pkg.added && depType === 'dev' && pnpmConfig?.saveDev === false && pnpmConfig?.extraEnv?.npm_command === 'add') {
```

Comment:
Here's how to check for the command name:

https://github.com/pnpm/pnpm/blob/73b936ff9559ffe8dafd5cee700c7c266688bffa/cli/default-reporter/src/index.ts#L41